uber/site_sections/registration.py

```diff
@@ -33,6 +33,14 @@ def checking_at_the_door(self, *args, **kwargs):
 @all_renderable(c.PEOPLE, c.REG_AT_CON)
 class Root:
     def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
+        # DEVELOPMENT ONLY: it's an extremely convenient shortcut to show the first page
+        # of search results when doing testing. it's too slow in production to do this by
+        # default due to the possibility of large amounts of reg stations accessing this
+        # page at once. viewing the first page is also rarely useful in production when
+        # there are thousands of attendees.
+        if c.DEV_BOX and (not page or page == 0):
```

Comment:
You should instead say

```python
if c.DEV_BOX and not int(page):
```

This is better for two reasons:

1) because ``0`` is falsey, you don't need a separate check for that

2) Because ``'0'`` is NOT falsey, so your code as written wouldn't work for that case